src/modules/client/components/clientAlerts/ClientAlert.tsx

```diff
@@ -0,0 +1,90 @@
+import { Alert, AlertTitle, Box, Typography } from '@mui/material';
+import theme from '@/config/theme';
+import {
+  clientNameAllParts,
+  parseAndFormatDate,
+  parseAndFormatDateTime,
+} from '@/modules/hmis/hmisUtil';
+import {
+  ClientAlertFieldsFragment,
+  ClientWithAlertFieldsFragment,
+} from '@/types/gqlTypes';
+
+export interface ClientAlertProps {
+  alert: ClientAlertFieldsFragment;
+  client: ClientWithAlertFieldsFragment;
+  shouldShowClientName?: boolean;
```

Comment:
`showClientName` or `hideClientName` would be more consistent with other props in the codebase